### ISSUE TITLE
Namespace pollution with `seq` function

### ISSUE THREAD

**Issue by apoliakov** [Paradigm4/SciDBR#231]
This simple sequence fails

```
$ cat db_test.R
#library('scidb')

db = scidb::scidbconnect(
  username = 'scidbadmin',
  password = readLines('~/.scidb_pass'),
  host = 'localhost',
  protocol = 'https',
  port= 8083
)

x = c(1,2,3,4)
y = scidb::as.scidb(db,x, types = c('int64'), attrs=c('chrom'), gc=T)


$ Rscript db_test.R
Error in seq.default(1, total_length, block_size) :
  'by' must be of length 1
Calls: <Anonymous> ... matvec2scidb -> multipart_post_load -> seq -> seq.default
Execution halted
```

It works if you uncomment the "library('scidb')" in the file. That means we got a nasty namespace pollution somewhere. We should fix. SciDB should be usable without having to load_library(). It's likely we are using some other version of "seq" or something it calls. Some inner function should be prepended with ::

We should avoid these kinds of "namespace pollution" issues to the extent possible. "library(scidb)" should not change how R behaves.

**Comment by apoliakov** [comment]
@ksens @arko-bhattacharya hey guys take a look - did you run into this before?

**Comment by arko-bhattacharya** [comment]
Hi Alex, I am looking into it.

**Comment by arko-bhattacharya** [comment]
This error occured because - 
* while uploading vectors in a multi-part fashion the code restores to a scidb option to get the maximum byte size of each block.
* as scidb was not loaded this option was unavailable to matvec2scidb and it couldn't create the length of each block of vector to upload to SciDB.

**Comment by apoliakov** [comment]
Thank you @arko-bhattacharya !

**Comment by ksens** [comment]
@apoliakov 

This issue should be fixed with the latest PR #232 by @arko-bhattacharya . Do take the latest version of scidbR for a spin and let us know if you still face any issues.
